include/config/config_rom.h

```diff
@@ -6,7 +6,7 @@
 
 // Internal ROM name. NEEDS TO BE **EXACTLY** 20 CHARACTERS. Can't be 19 characters, can't be 21 characters. You can fill it with spaces.
 // The end quote should be here:               "
-#define INTERNAL_ROM_NAME "SUPER MARIO 64      "
+#define INTERNAL_ROM_NAME "HackerSM64          "
```

Comment:
I'm cool with this, anyone else's thoughts?

Reply:
Oh right, for an explanation, iirc VC will break if we don't rename the rom lol

Reply:
Fine by me too